sympy/core/power.py

```diff
@@ -1444,11 +1444,13 @@ def _eval_nseries(self, x, n, logx):
 
                 # express "rest" as: rest = 1 + k*x**l + ... + O(x**n)
                 rest = expand_mul((b - prefactor)/prefactor)
+                rest = rest.simplify()
```

Comment:
I think that there was only one test where this was needed: `test_issue_6364`. Maybe that should be commented here in case someone would want to experiment with cheaper variants of `simplify`. I don't think that even `simplify` would be too expensive. In most cases, `rest` will be 0 or a symbol with trivial `_eval_simplify`.

Reply:
so shall i just comment test_issue_6364